src/decisionengine/framework/engine/DecisionEngine.py

```diff
@@ -570,13 +651,16 @@ def main(args=None):
     If args is a list, it will be used instead of sys.argv (for unit testing)
     """
     options = parse_program_options(args)
+    _check_metrics_env(options)
     global_config_dir = policies.global_config_dir()
     global_config, channel_config_loader = _get_de_conf_manager(
         global_config_dir, policies.channel_config_dir(), options
     )
     try:
+        # app.run()
```

Comment:
should this be commented out?

Reply:
Actually, during the massive rebase I just did, I thought that was leftover from the main branch, but I'm mistaken on that.  I'll get rid of that.